.ci/scripts/convert_tap_to_junit.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Is this file a candidate now to be part of the shared library?

Reply:
That's a good point, I'll raise an issue to discuss it then